rotkehlchen/api/rest.py

```diff
@@ -4119,7 +4119,11 @@ def add_evm_transaction_by_hash(
                 ),
                 status_code=status_code,
             )
-        evm_manager.transactions_decoder.decode_transaction(transaction=transaction, tx_receipt=tx_receipt)  # noqa: E501
+
+        evm_manager.transactions_decoder.decode_transaction_hashes(
+            ignore_cache=True,
```

Comment:
What happened before? We were doing nothing if it was already decoded?

Reply:
No, it was calling `decode_transaction` that always performed a detection